### PR TITLE
Fixes #51 Describe snapshots

### LINKED ISSUE #42: Add rabbit
Include rabbit into the list of available species (introduced in 7.1)

### LINKED ISSUE #45: Include pregnancy population
Include pregnancy population in the list of available populations (introduced in 7.2)

### LINKED ISSUE #51: Describe snapshots
Describe the snapshot-feature (introduced in 7.3)

### LINKED ISSUE #52: Describe "user defined parameters"
Describe "user defined parameters" feature (introduced in 7.3)

### LINKED ISSUE #54: Describe "Intestinal solubility as table function of pH"
Describe "Intestinal solubility as table function of pH" (introduced in 7.3)

### PR DESCRIPTION
**Pull request by Yuri05**
Fixes #42 Add rabbit
Fixes #45 Include pregnancy population
Fixes #54 Describe "Intestinal solubility as table function of pH"
Fixes #52 Describe "user defined parameters"

Plus a lot of small fixes.

### CODE REVIEW

File: part-3/importing-exporting-project-data-models.md [hunk 1]
```diff
@@ -113,3 +113,38 @@ and select **Export To CSV...**
 - If you have created a population simulation you can also export the population data by right mouse click on the simulation name within the **Simulations Explorer** and select **Export To CSV...**
 
 A window opens where you can choose the directory and enter the \*.csv file name.
+
+## Exporting Project to Snapshot / Loading Project from Snapshot
+
```
Comment (Yuri05) [inline part-3/importing-exporting-project-data-models.md]: Description taken from 7.3 release notes https://github.com/Open-Systems-Pharmacology/Suite/releases/tag/v7.3.0

File: part-3/pk-sim-compounds-definition-and-work-flow.md [hunk 2]
```diff
@@ -117,6 +117,12 @@ The solubility of the compound is only needed for the oral administration route.
 First estimates can be made using water solubility. However, especially for lipophilic compounds this value might underestimate the solubility in the intestine so that it is better to use a value obtained under bio-relevant conditions (e.g. in Fasted State Simulated Intestinal Fluid, FaSSIF). If different values are available for one compound (e.g. in FaSSIF and in Fed State Simulated Intestinal Fluid, FeSSIF), several alternative solubility values can specified and the appropriate value can then chosen in the **Simulation**.
 {% endhint %}
 
+#### Intestinal solubility as table function of pH
+Intestinal solubility can also be defined as a linear interpolation of measured (pH, Solubility) data pairs.
```
Comment (Yuri05) [inline part-3/pk-sim-compounds-definition-and-work-flow.md]: Description taken from 7.3 rlease notes https://github.com/Open-Systems-Pharmacology/Suite/releases/tag/v7.3.0

**Comment by Yuri05** [comment]
Apart from some new functionality described, most of the fixes are formatting, broken references, old support adress etc.

File: part-2/getting-started.md [hunk 3]
```diff
@@ -2,11 +2,15 @@
...
-| Processor  | minimum 1 GHz (the faster, the better) |     |     |     |
-| Memory     | 2 GB RAM, 3+ GB recommended            |     |     |     |
-| Disk space | minimum 2 GB                           |     |     |     |
+| OS         | Windows 7®, Windows 8®, Windows 10®, Windows Server 2008 R2®, Windows Server 2012 R2®, Windows Server 2016®    |
+| ---------- | -------------------------------------- |
+| Processor  | minimum 1 GHz (the faster, the better) |
+| Memory     | 2 GB RAM, 3+ GB recommended            |
+| Disk space | minimum 2 GB                           |
+||              **Optional software**                 ||
+| Matlab®    | version 2013b or later - 32bit         |
```
Comment (msevestre) [inline part-2/getting-started.md]: We are going to have to change that very soon as well

File: part-2/getting-started.md [hunk 4]
```diff
@@ -36,26 +40,24 @@ MoBi® Toolbox for Matlab® is currently only working with the 32- bit version o
 
 To install the Open Systems Pharmacology Suite core components:
 
-1.  Obtain installation packages, either distributed via USB flash drive or to be downloaded from https://github.com/Open-Systems-Pharmacology
-
-    An internet connection is required as software components such as .NET Framework (in case the required version is not installed on the computer) may need to be downloaded from server.
+1.  Obtain installation packages from http://setup.open-systems-pharmacology.org/
```
Comment (msevestre) [inline part-2/getting-started.md]: obtain??? Who  says that!
Download

**Pull request merged**